src/com/maddyhome/idea/vim/KeyHandler.java

```diff
@@ -753,8 +763,10 @@ private void startWaitingForArgument(Editor editor,
         // the key handler when it's complete.
         if (action instanceof InsertCompletedDigraphAction) {
           editorState.startDigraphSequence();
+          setPromptCharacterEx('?');
```

Comment:
Is this code supposed to insert `?` character under the caret when the user inserts digraph? IIRC, this functionality already works, isn't it?

Reply:
Yes, but using KeyHandler changed the logic and I needed to implement it here